docs/src/misc.rst

```diff
@@ -271,6 +271,21 @@ API
     and :man:`inet_pton(3)`. On success they return 0. In case of error
     the target `dst` pointer is unmodified.
 
+.. c:macro:: UV_IF_NAMESIZE
+
+    Maximum IPv6 interface identifier name lenght.  Defined as
+    `IFNAMSIZ` in Unix and `IF_NAMESIZE` in Linux and Windows.
+
+    .. versionadded:: 1.14.0
+
+.. c:function:: char* uv_if_indextoname(unsigned int ifindex, char* ifname)
+
+    Cross-platform IPv6-capable implementation of :man:`if_indextoname(3)`.
+    Writes the result to `ifname`, which must be at least `UV_IF_NAMESIZE` long.
+    On success returns the generated string.  In case of error returns `NULL`.
```

Comment:
I tried to describe the error codes.  I could not decipher from the documentation what Windows does in various error conditions.